docs/manual.md

```diff
@@ -1122,8 +1153,27 @@ cardinality does not require `2^|NUMS|` constraints, when using a recursive
 operator.
 
 
+<a name="apalacheMod"></a>
+# 9. The Apalache module
+
+Similar to the `TLC` module, we provide the module called `Apalache`, which can
+be found in
+[src/tla](https://github.com/informalsystems/apalache/tree/unstable/src/tla).
+Most of the operators in that modules are introduce internally by Apalache,
+when it is rewriting a TLA+ specification.  It is useful to read the comments
+to the operators defined in `Apalache.tla`, as they will help you in
+understanding the [detailed output](#detailed) produced by the tool, see.
```

Comment:
Was "see" here meant to be followed to a reference to something?

Reply:
@konnov I think this question may have gotten hidden by Github eliding some of the comments.